docs/content/policy-language.md

```diff
@@ -1124,24 +1125,43 @@ no_bitcoin_miners_using_negation with apps as [{"name": "bitcoin-miner"}, {"name
 ```live:eg/data/correct_negation/2:output:expect_undefined
 ```
 
-> The `undefined` result above is expected because we did not define a default
-> value for `no_bitcoin_miners_using_negation`. Since the body of the rule fails
-> to match, there is no value generated.
+{{< info >}}
+The `undefined` result above is expected because we did not define a default
+value for `no_bitcoin_miners_using_negation`. Since the body of the rule fails
+to match, there is no value generated.
+{{< /info >}}
 
 Alternatively, we can implement the same kind of logic inside a single rule
 using [Comprehensions](#comprehensions).
 
 ```live:eg/data/comprehesion_alternative:module:read_only
 no_bitcoin_miners_using_comprehension {
-    bitcoin_miners := {app | app := apps[_]; app.name == "bitcoin-miner"}
+    bitcoin_miners := {app | some app in apps; app.name == "bitcoin-miner"}
     count(bitcoin_miners) == 0
 }
 ```
 
-> Whether you use negation or comprehensions to express FOR ALL is up to you.
-> The comprehension version is more concise and does not require a helper rule
-> while the negation version is more verbose but a bit simpler and allows for
-> more complex ORs.
+By importing the future keyword "every", you get another option to express universal
+quantification:
+
+```live:eg/data/every_alternative:module:read_only
+import future.keywords.every
+
+no_bitcoin_miners_using_every {
+    every app in apps {
+        app.name != "bitcoin-miner"
+    }
+}
+```
+
+{{< info >}}
+Whether you use negation, comprehensions, or `every` to express FOR ALL is up to you.
+The comprehension version is more concise and does not require a helper rule
+while the negation version is more verbose but a bit simpler and allows for
+more complex ORs.
+Ideally, the `every` variant allows you to express something very closely aligned
```

Comment:
I'd think we want to encourage people to use `every` in most cases, no? I think we should be more prescriptive here.

Reply:
Perhaps we should mention that `every` has the same semantics as NOT-SOME-NOT rewriting?